test/hotspot/jtreg/runtime/CommandLine/VMDeprecatedOptions.java

```diff
@@ -62,6 +67,9 @@ public class VMDeprecatedOptions {
             {"CreateMinidumpOnCrash", "false"}
           }
         ));
+        if (wb.isJFRIncluded()) {
+            deprecated.add(new String[] {"FlightRecorder",                  "false"});
```

Comment:
No need for all the extra whitespace after the comma - just a single space please.

Reply:
That's right. I fixed it.